### ISSUE TITLE
Multiple zeniths

### ISSUE THREAD

**Issue by LukasNickel** [cta-observatory/lstmcpipe#190]
We should make it possible to run on multiple zeniths at once instead of starting multiple PRs.
Not super trivial in the current structure, but will be needed at some point anyway

**Comment by chaimain** [comment]
As we will soon be having 'all-sky' MC, we will be having MC for various sky paintings, and not just zenith :)

**Comment by LukasNickel** [comment]
True, even worse, for us in terms of work that is :D

**Comment by LukasNickel** [comment]
We will need to wait for the grid simulations and more information from discussions in the coming weeks


> various sky pAintings

Sounds cool as well

**Comment by vuillaut** [comment]
> We should make it possible to run on multiple zeniths at once instead of starting multiple PRs. Not super trivial in the current structure, but will be needed at some point anyway

Yes, but maybe not in the context of PR requests (not at first at least).
With the sky-covered MC, we should have less specific requests for productions and we should focus on the _official_ productions first.
